input/pagecontent/2-8-Extensions.md

```diff
@@ -1,20 +1,20 @@
-The specification is not prescriptive about support for extensions. However, to support extensions, the specification reserves the name `extension` and will never define an element with that name, allowing implementations to use it to provide custom behavior and information. The value of an extension element MUST be a pre-coordinated JSON object. For example, an extension on a notification could look like this:
-
-```json
-{
- "context": [{
-   "key": "patient",
-   "resource": {
-    "resourceType": "Patient",
-    "id": "ewUbXT9RWEbSj5wPEdgRaBw3"
-   }
-  },
-  {
-   "key": "extension",
-   "data": {
-    "user-timezone": "+1:00"
-   }
-  }
- ]
-}
-```
+The specification is not prescriptive about support for extensions. However, to support extensions, the specification reserves the name `extension` and will never define an element with that name, allowing implementations to use it to provide custom behavior and information. The value of an extension element MUST be a valid JSON object. For example, an extension on a notification could look like this:
```

Comment:
```suggestion
The specification is not prescriptive about support for extensions. However, to support extensions, the specification reserves the name `extension` and will never define an element with that name, allowing implementations to use it to provide custom behavior and information. The value of an extension element MUST be a valid JSON object and is typical pre-coordinated. For example, an extension on a notification could look like this:
```

Hey Paul, 

Agree that there oughtn't be a normative requirement for pre-coordination.

There's value in setting the expectation that extensions can't be meaningfully used unless they are also pre-coordinated, right?

Reply:
Hi Isaac,

I can live with your suggestion. But make it grammatically correct "typical" -> "typically".